src/app/color-picker.ts

```diff
@@ -180,7 +183,7 @@ class ColorPicker extends HTMLElement {
     const scope = this;
     this.state = new Proxy(scope.stateDefaults, {
       // eslint-disable-next-line
-      set(target: IColorPickerState, key: string | symbol, value: any, _receiver: any): boolean {
+      set(target: IColorPickerState, key: keyof IColorPickerState, value: any, _receiver: any): boolean {
```

Comment:
This is an unrelated change but protects against using an invalid key name below.